packages/jest-runner/src/index.ts

```diff
@@ -57,10 +65,11 @@ class TestRunner {
   async runTests(
```

Comment:
I believe this is the only unresolved thing. This should work

```ts
async runTests(
  tests: Array<JestTest>,
  watcher: JestTestWatcher,
  onStart: JestOnTestStart | undefined,
  onResult: JestOnTestSuccess | undefined,
  onFailure: JestOnTestFailure | undefined,
  options: JestTestRunnerOptions,
)
```

Then remove them in Jest 27

Reply:
Oh I had just added the comment above. posting again for easy visibility

---

I can create another function `verboseRunTests()` (or something similar) which would take `onStart` and other params. Thus leaving `runTests()` unchanged

Should I proceed with this? It depends how soon will we see Jest 27 ig :grin:

Reply:
Just making them `| undefined` seems easier to me than adding a new API surface.